testsuite/replay_verify.py

```diff
@@ -16,47 +16,47 @@
 
 
 TESTNET_RANGES = [
-    [255584107, 271874718],
-    [271874719, 300009463],
-    [300009464, 324904819],
-    [324904820, 347234877],
-    [347234878, 366973577],
-    [366973578, 399489396],
-    [399489397, 430909965],
-    [430909966, 449999999],
-    [450000000, 462114510],
-    [462114511, 478825432],
-    [478825433, 483500000],
-    [483500001, 516281795],
-    [516281796, 551052675],
-    [551052676, 582481398],
-    [582481399, 640_000_000],
-    [640_000_001, 980_000_000],
-    [980_000_000, 1_100_000_000],
-    [1_100_000_001, 1_200_000_000],
-    [1_200_000_001, sys.maxsize],
+    (962_000_000, 978_000_000),
+    (978_000_000, 994_000_000),
+    (994_000_000, 1_009_000_000),
+    (1_009_000_000, 1_025_000_000),
+    (1_025_000_000, 1_041_000_000),
+    (1_041_000_000, 1_057_000_000),
+    (1_057_000_000, 1_073_000_000),
+    (1_073_000_000, 1_088_000_000),
+    (1_088_000_000, 1_104_000_000),
+    (1_104_000_000, 1_120_000_000),
+    (1_120_000_000, 1_136_000_000),
+    (1_136_000_000, 1_151_000_000),
+    (1_151_000_000, 1_167_000_000),
+    (1_167_000_000, 1_183_000_000),
+    (1_183_000_000, 1_199_000_000),
+    (1_199_000_000, 1_215_000_000),
+    (1_215_000_000, 1_230_000_000),
+    (1_230_000_000, 1_246_000_000),
+    (1_246_000_000, 1_262_000_000),
 ]
 
 MAINNET_RANGES = [
-    [155_000_001, 180_000_000],
-    [180_000_001, 190_000_000],
-    [190_000_001, 200_000_000],
-    [200_000_001, 215_000_000],
-    [215_000_001, 225_000_000],
-    [225_000_001, 235_000_000],
-    [235_000_001, 246_000_000],
-    [246_000_001, 270_000_000],
-    [270_000_001, 295_000_000],
-    [295_000_001, 321_000_000],
-    [321_000_001, 331_000_000],
-    [331_000_001, 354_000_000],
-    [354_000_001, 400_000_000],
-    [400_000_001, 490_000_000],
-    [490_000_000, 550_000_000],
-    [550_000_001, 600_000_000],
-    [600_000_001, 640_000_000],
-    [948_918_390, 950_000_000],
-    [950_000_001, sys.maxsize],
+    (392_000_000, 408_000_000),
+    (408_000_000, 424_000_000),
+    (424_000_000, 439_000_000),
+    (439_000_000, 455_000_000),
+    (455_000_000, 471_000_000),
+    (471_000_000, 487_000_000),
+    (487_000_000, 503_000_000),
+    (503_000_000, 518_000_000),
+    (518_000_000, 534_000_000),
+    (534_000_000, 550_000_000),
+    (550_000_000, 566_000_000),
+    (566_000_000, 581_000_000),
+    (581_000_000, 597_000_000),
+    (597_000_000, 613_000_000),
+    (613_000_000, 629_000_000),
+    (629_000_000, 640_000_000),
```

Comment:
It would be great if we could somehow generate this list (say have some skipped ranges that are subtracted somehow / truncated?)

Or even just leave a comment so its obvious what / why we're skipping for other people reading this